package.json

```diff
@@ -50,8 +49,12 @@
     "uuid": "^8.3.2",
     "xml2js": "^0.4.23",
     "yargs": "^17.5.1",
+    "yn": "^3.1.1",
```

Comment:
This is used at runtime but wasn't in package.json, I think we were getting it transitively from isolated-vm. The latest version of this package is 5.0.0, but that version uses a different module style that is not compatible with our code. So pegged this to the same version we were already using. When dependabot tries to update this, hopefully something will fail so we don't accidentally upgrade it...